src/style.css

```diff
@@ -1 +1,25 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  margin-right: 4px;
```

Comment:
do not set margin-right for last star in the row

Reply:
In tests 7 passed 0 failed. Maybe the problem is something else?